packages/@react-spectrum/s2/src/Checkbox.tsx

```diff
@@ -66,8 +66,8 @@ const wrapper = style({
 
 export const box = style<RenderProps>({
   ...focusRing(),
-  size: 'control-sm',
-  borderRadius: 'control-sm',
+  ...controlBorderRadius('sm'),
+  size: controlSize('sm'),
```

Comment:
a little confused so far on when it's a spreadable property vs an individual property, the name looks like they should both be one or the other

Reply:
just the return type of the function. If it returns multiple properties it has to be spread.